install.js

```diff
@@ -28,7 +28,7 @@ if (platform === 'linux') {
     platform += '32'
   }
 } else if (platform === 'darwin') {
-  platform = 'mac32'
+  platform = 'mac64'
```

Comment:
@chaseadamsio there is no mac32 anymore. Then what, we just fail the installation?